scraper.rb

```diff
@@ -45,15 +59,29 @@ def scrape_list(url)
       phone: tds[5].xpath('.//i[@class="icon-bell"]//following-sibling::text()[contains(.,"+54")]').text.tidy,
       facebook: tds[5].css('a[href*="facebook"]/@href').text.tidy,
       twitter: tds[5].css('a[href*="twitter"]/@href').text.tidy,
-      # 6 year terms, so we should split these all into three, but for
-      # now just take the latest one
-      term: 2015,
       source: person_url,
     }
     # }.merge(scrape_person(person_url))
     data[:image] = URI.join(url, data[:image]).to_s unless data[:image].to_s.empty?
-    ScraperWiki.save_sqlite([:id, :term], data)
+    memberships << data
   end
+  memberships
+end
+
+memberships_from_page = scrape_list('http://www.senado.gov.ar/senadores/listados/listaSenadoRes')
+
+data = CombinePopoloMemberships.combine(
+  id: memberships_from_page,
+  term: terms,
+)
+
+# If the start and end dates would be implied by the term anyway, set
+# them to NULL.
```

Comment:
If you're doing this step for EP reasons, it's not necessary, as the EP build process will do this anyway.